tests/PatternAssertions/BasePatternAssert.php

```diff
@@ -168,4 +168,23 @@ protected function assertElementNotExists(string $selector, Crawler $crawler): v
     ));
   }
 
+  /**
+   * Asserts the image of the pattern.
+   *
+   * @param array|null $expected_image
```

Comment:
I have not understood why type is set as `array|null` for all expected values. If we expect value then it shouldn't be NULL, from my point of view.